src/GraphQL/Reflection/AccessorFieldResolver.cs

```diff
@@ -7,9 +7,9 @@ namespace GraphQL.Reflection
     internal class AccessorFieldResolver : IFieldResolver
     {
         private readonly IAccessor _accessor;
-        private readonly IDependencyResolver _dependencyResolver;
+        private readonly IServiceProvider _dependencyResolver;
 
-        public AccessorFieldResolver(IAccessor accessor, IDependencyResolver dependencyResolver)
+        public AccessorFieldResolver(IAccessor accessor, IServiceProvider dependencyResolver)
```

Comment:
I think it is worth renaming all the arguments with `dependencyResolver` name to `serviceProvider` for the purpose of not mentioning now obsolete interface `IDependencyResolver` and thereby not confusing users